sys-fs/bees/files/v0.5-gentoo_build.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/localconf b/localconf
```

Comment:
Have you submitted this patch upstream?

Reply:
No, this patch should not be upstream. But I could instead create this file from scratch. Actually, I'm the one who added the possibility to have support for a localconf file upstream.